packages/design-system/src/components/Button/Button.stories.jsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable react/prop-types */
```

Comment:
I wonder if we can disable this at the `.eslintrc` level for all `*.stories.jsx` so we don't have to copy this for all stories.

Reply:
Updated!